Add German translation

daPhipz opened the pull request:
As the title says.
I also made Yannick's translation actually a French translation - or should it be `fr_CH`? I am half French myself and I couldn't find any Swiss specialities in his translation. That's why I used `fr` and not `fr_CH`.

ymauray commented:
Well spotted for the mistake in `fr.po`! Can you tell I've been copying and pasting stuff manually? :P

Can you please update the PR by adding `de` to the list of supported locales in `lib/src/app.dart`, around line 52?

Thanks in advance!

daPhipz commented:
> Can you tell I've been copying and pasting stuff manually?

Nooo, not at all! I would never do such things :innocent: 
I rebased my PR and added `de` to the file!

Ready to merge as far as I am concerned.

ymauray commented:
Thank you!!
